src/tests/tablet/schedule/CannotCreateMeetingWithoutSubject.java

```diff
@@ -0,0 +1,43 @@
+package tests.tablet.schedule;
+
+import static framework.common.AppConfigConstants.EXCEL_INPUT_DATA;
+
+import java.util.List;
+import java.util.Map;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import framework.pages.tablet.HomeTabletPage;
+import framework.pages.tablet.SchedulePage;
+import framework.utils.readers.ExcelReader;
+
+/**
+ * TC9: Verify a user cannot create a meeting if the subject is not specified
+ * @author Asael Calizaya
+ *
+ */
+public class CannotCreateMeetingWithoutSubject {
+
+	@Test(groups = "ACCEPTANCE")
+	public void testCannotCreateMeetingWithoutOrganizer() {
```

Comment:
without organizer or without subject??
your class is subject the test is organizer